ffbs-parker-nodeconfig/files/usr/sbin/noderoute.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+# simple script to loop noderoute lua script
+
+tmpdir=/tmp/ff-Ohb0ba0u/
+
+LOGGER="logger -s -t noderoute.sh"
+$LOGGER Starting up.
+
+check_batman() {
+    # check if we have a functional batman-setup
+    if ip l | grep -q "bat0:"; then
```

Comment:
For style, I would prefer having an early exit here instead of nesting the if's.

```suggestion
    if ! ip l | grep -q "bat0:"; then
        $LOGGER ERROR: No BATMAN-interface found
        return 1
    fi
```

remove the neting and remove lines 23-26 (they have been moved up).

Reply:
@SmithChart still open

Reply:
See https://github.com/SmithChart/community-packages/pull/8

Reply:
@grische can you verify that this has been resolved by https://github.com/SmithChart/community-packages/pull/8 ?